fhir-operation-bulkdata/src/main/java/com/ibm/fhir/operation/bulkdata/BulkDataConstants.java

```diff
@@ -31,6 +36,10 @@
     public static final String PARAM_GROUP_ID = "groupId";
     public static final String PARAM_JOB = "job";
 
+    // Encryption key used for JavaBatch Job ID
+    public static final SecretKeySpec BATCHJOBID_ENCRYPTION_KEY =
+            BulkDataConfigUtil.getBatchJobIdEncryptionKey(FHIRServerUtils.getJNDIValue("config/bulkDataBatchJobIdEncryptionKey", null));
```

Comment:
Pull this value from the tenant configuration for bulkdata or pull from the property group.

Reply:
I don't think this need to be tenant specific. so I didn't put it into bulkdata.json or any of then per tenant config file. I considered server.xml or server.env, and thought server.xml is the better place to put this tenant independent config.

Reply:
changed to use fhir server config.